modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -47,22 +48,24 @@ You can use the information in this topic to publish Mule 3 and Mule 4 assets in
 If you are publishing your asset to the master organization, the group ID is the same as the organization ID.
 The URL format is:
 +
-`+https://maven.anypoint.mulesoft.com/api/v1/organizations/ORGANIZATION_ID/maven+`
+`+https://maven.anypoint.mulesoft.com/api/v3/organizations/ORGANIZATION_ID/maven+`
 +
 In the EU, use:
 +
-`+https://maven.eu1.anypoint.mulesoft.com/api/v1/organizations/ORGANIZATION_ID/maven+`
+`+https://maven.eu1.anypoint.mulesoft.com/api/v3/organizations/ORGANIZATION_ID/maven+`
 +
 If you are publishing your asset to a business group, the group ID is the same as the business group ID.
 The URL format is:
 +
-`+https://maven.anypoint.mulesoft.com/api/v1/organizations/BUSINESS_GROUP_ID/maven+`
+`+https://maven.anypoint.mulesoft.com/api/v3/organizations/BUSINESS_GROUP_ID/maven+`
 +
 In the EU, use:
 +
-`+https://maven.eu1.anypoint.mulesoft.com/api/v1/organizations/BUSINESS_GROUP_ID/maven+`
+`+https://maven.eu1.anypoint.mulesoft.com/api/v3/organizations/BUSINESS_GROUP_ID/maven+`
 +
-. Add the asset type in properties part in your project's POM file. For example:
+. In your project's POM file, add the asset type in the properties section.
```

Comment:
I would prefer to emphasize that the asset type could be inferred by the classifier of the file. Like: https://github.com/mulesoft-labs/exchange-documentation-samples/blob/master/example/pom.xml#L36 or https://github.com/mulesoft-labs/exchange-documentation-samples/blob/master/mule-app/pom.xml#L35

But it still could be defined by the asset type like this example: https://github.com/mulesoft-labs/exchange-documentation-samples/blob/master/custom-asset/pom.xml#L17

This repository has examples about how to publish any typical maven asset types: https://github.com/mulesoft-labs/exchange-documentation-samples. I would add this since it is not easy to publish using maven for all the asset types and having examples of each one could be very helpful.

Reply:
Much better! I added a link to the repository, and changed the example to match https://github.com/mulesoft-labs/exchange-documentation-samples/blob/master/example/pom.xml . It may still be useful to go over all of this file with you to make sure it uses the best examples, emphasizes the right things, and leads the user in the right way.

Reply:
agree, let's review it together